components/home/HeroFullImage.tsx

```diff
@@ -0,0 +1,196 @@
+import { useTheme } from 'common/theme';
```

Comment:
Is there an old `HeroFullImage.js` which should be deleted?

Reply:
Good catch - I thought I used rename but it's still lingering there